lib/topological_inventory/ansible_tower/messaging_client.rb

```diff
@@ -11,14 +11,13 @@ def worker_listener
         @worker_listener ||= ManageIQ::Messaging::Client.open(worker_listener_opts)
       end
 
-      def targeted_refresh_listener
-        @targeted_refresh_listener ||= ManageIQ::Messaging::Client.open(targeted_refresh_listener_opts).tap do |client|
-          # persistent workers by pod hostname, this will prevent rebalances.
-          client.send(:kafka_client)[:"group.instance.id"] = ENV['HOSTNAME']
```

Comment:
Could this caused current frozen prod pods? 
Anyway it didn't solve the problem so I'm reverting it

Reply:
Yeah I'm not sure, this should just prevent rebalances, but obviously it didn't work so I'm ok reverting it.